packages/snap/src/features/homepage/components/CustodianList.tsx

```diff
@@ -39,10 +41,12 @@ const custodianLogos: Record<string, string> = {
 
 type CustodianListProps = {
   accounts?: CustodialKeyringAccount[];
+  devMode?: boolean;
```

Comment:
we show the toggle inside the custodian list component, so as not to break up the form